### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by raiz0x** [community/community#128935] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi,

I have two questions:

1. It's possible to to make a file/folder or project private? Private for everyone excluding me, but to have external access(via secret link hash/key api) to access files(raw) with direct web/curl.
2. It's possible to use http for accesing files(raw) too? For now I'm redirected to https.

**Comment by queenofcorgis** [comment, marked as answer]
Hey @raiz0x - these were interesting questions, I appreciate you asking. 

1. There is no way to do these via repository settings, but this [post](https://24ways.org/2013/keeping-parts-of-your-codebase-private-on-github/) may be of help. 
2.  Could you explain more about what you're trying to do here so the community can better assist?

**Reply by raiz0x** [reply]
Hey @queenofcorgis 

Thanks for reply. About 2nd question, I have some apps which can't use SSL(https) and I have to load site under non-SSL(http). For me could be better if I could use github as primary `cloud`(storage) where to access some files(in raw format) under non-SSL url.

**Reply by queenofcorgis** [reply]
Thanks for expanding further - GitHub enforces HTTPS and will redirect all HTTP requests to HTTPS.
